lms/djangoapps/certificates/models.py

```diff
@@ -213,6 +215,8 @@ def get_queryset(self):
 class GeneratedCertificate(models.Model):
     """
     Base model for generated certificates
+
+    .. no_pii: Information that may be PII in a cert is stored and retired elsewhere
```

Comment:
I'm also confused why this isn't just a positive annotation.

Reply:
There is no PII stored here, the comment is just to explicitly tell people that if they are looking for where Certs pull PII from this isn't it. I can update it to point at the other places more explicitly if I can find them.